src/intrinsics/fb-www/global.js

```diff
@@ -9,13 +9,46 @@
 
 /* @flow */
 
+import { parseExpression } from "babylon";
 import type { Realm } from "../../realm.js";
-import { AbstractValue, NativeFunctionValue, Value, StringValue } from "../../values/index.js";
+import { AbstractValue, NativeFunctionValue, Value, ObjectValue, StringValue } from "../../values/index.js";
 import buildExpressionTemplate from "../../utils/builder.js";
 import { createMockReact } from "./react-mocks.js";
 import { createMockReactRelay } from "./relay-mocks.js";
+import { Environment } from "../../singletons.js";
 import invariant from "../../invariant";
 
+// Based on www babelHelpers fork.
+const babelHelpersCode = `
+{
+  inherits(subClass, superClass) {
+    Object.assign(subClass, superClass);
+    subClass.prototype = Object.create(superClass && superClass.prototype);
+    subClass.prototype.constructor = subClass;
+    subClass.__superConstructor__ = superClass;
```

Comment:
What does `__superConstructor__` even do? Every other polyfill does a `__proto__` assignment or `setPrototypeOf` here instead.

Reply:
That's an FB specific thing, I don't actually remember what it does. Need to check.

Reply:
If you change that one line to `__proto__`, everything works as expected. but because we are setting it to be `superClass`, it tries to emit the function `React.Component` instead; which I guess is right here.